spec/src/main/asciidoc/ch08-entity-packaging.adoc

```diff
@@ -168,149 +170,163 @@ Examples:
 
 ===== name
 
-The _name_ attribute defines the name for the
-persistence unit. This name may be used to identify a persistence unit
-referred to by the _PersistenceContext_ and _PersistenceUnit_
-annotations and in the programmatic API for creating an entity manager
-factory.
+The _name_ attribute defines the name of the persistence unit. This name is
+used to identify the persistence unit referred to by a _PersistenceContext_
+or _PersistenceUnit_ annotation and in the programmatic API for creating an
+entity manager factory.
 
 ===== transaction-type [[a12296]]
 
-The _transaction-type_ attribute is used to
-specify whether the entity managers provided by the entity manager
-factory for the persistence unit must be JTA entity managers or
-resource-local entity managers. The value of this element is _JTA_ or
-_RESOURCE_LOCAL_. A _transaction-type_ of _JTA_ assumes that a JTA data
-source will be provided—either as specified by the _jta-data-source_
-element or provided by the container. In general, in Jakarta EE
-environments, a _transaction-type_ of _RESOURCE_LOCAL_ assumes that a
-non-JTA datasource will be provided. In a Jakarta EE environment, if this
-element is not specified, the default is _JTA_. In a Java SE
-environment, if this element is not specified, the default is
-_RESOURCE_LOCAL_.
+The _transaction-type_ attribute specifies whether entity managers created by
+the entity manager factory for the persistence unit are JTA entity managers or
+resource-local entity managers. The value of this element must be _JTA_ or
+_RESOURCE_LOCAL_:
+
+- _JTA_ means that a JTA data source is provided—either as specified by the
+  _jta-data-source_ element, or by the container.
+- In a Jakarta EE environment, _RESOURCE_LOCAL_ usually means that a non-JTA
+  datasource is provided.
+
+See <<a12302>>.
```

Comment:
this should probably contain some description of the pointer to understand a context about why it is here. Right now it is unclear why should one go there and what one can find there. Something like _For more details about data sources, see ..._ or _See ... for data source configuration_. From a different angle - does the pointer have to be here? I'd say it is good to have it

Reply:
I agree.

Reply:
@lukasj is that better?

Reply:
yes, thanks